src/main/java/org/sunbird/common/util/ProjectUtil.java

```diff
@@ -14,11 +19,16 @@
 public class ProjectUtil {
 
     public static CbExtLogger logger = new CbExtLogger(ProjectUtil.class.getName());
+    private static Pattern pattern;
+    public static final String EMAIL_PATTERN =
+            "^[_A-Za-z0-9-\\+]+(\\.[_A-Za-z0-9-]+)*@"
+                    + "[A-Za-z0-9-]+(\\.[A-Za-z0-9]+)*(\\.[A-Za-z]{2,})$";
 
     public static PropertiesCache propertiesCache;
     private static final ObjectMapper mapper = new ObjectMapper();
 
     static {
+        pattern = Pattern.compile(EMAIL_PATTERN);
```

Comment:
this is not required. Need to remove static block. Need to analyse again the need of this class. Can we move these to existing Util class ?